portfolio/src/main/webapp/index.html

```diff
@@ -83,21 +83,15 @@
   </div>
   </div>
   <div class="footer-container">
-    <div class="column1">
-      <a class="footer-item" href="#about-container">About</a>
-      <a class="footer-item" href="#projects-container">Projects</a>
-      <a class="footer-item" href="#work-container">My Work</a>
-      <a class="footer-item" href="#footer-container">Contact</a>
-    </div>
-    <div class="column2">
-      <p>Contact Me!</p>
-      <p class="form-message">We'll be in touch!</p>
-      <input type="text" name="" placeholder="First Name">
-      <input type="text" name="" placeholder="Last Name">
-      <input type="text" name="" placeholder="Email">
+    <div class="contactForm">
+      <p>Say Hi!</p>
+      <input class="form-Name" type="text" name="" placeholder="Name">
```

Comment:
Don't mix casing here. If you're using `kebab-case` then these should be `form-name` and `form-email`

Reply:
got it sticking to kebab for now

Reply:
done